src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -531,12 +519,13 @@ void BLEManagerImpl::DriveBLEState(void)
     // Otherwise, stop advertising if it is enabled.
     else if (mFlags.Has(Flags::kAdvertising))
     {
+        SILABS_LOG("Stop Advertising");
         err = StopAdvertising();
         SuccessOrExit(err);
     }
 
 exit:
-    ChipLogProgress(DeviceLayer, "DriveBLEState End");
+    SILABS_LOG("%s End", __func__);
```

Comment:
excessive logging. It is only helpful for active debugging. I suggest removing it.

Reply:
Removed